CTOR-899 : Refresh network-cisco-standard-ssh (#3574)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-ssh.md

1. Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
2. Complétez les champs **Nom**, **Alias** & **IP Address/DNS** correspondant à votre ressource.
3. Appliquez le modèle d'hôte **Net-Cisco-Standard-SSH-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.

| Macro           | Description                                                                                                                                                          | Valeur par défaut | Obligatoire |
|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| SSHUSERNAME     | Define the user name to log in to the host                                                                                                                           |                   |             |

> | SSHPASSWORD     | Define the password associated with the user name. Cannot be used with the sshcli backend. Warning: using a password is not recommended. Use --ssh-priv-key instead  |                   |             |

Author: Si on recommande d'utiliser `--ssh-priv-key` on devrait peut-être changer le template pour ajouter la macro ce serait plus simple pour l'utilisateur ?
@omercier

Author: Vérifier si mettre 2x l'option `--ssh-priv-key` peut induire un problème ou pas => si oui breaking change possible et il faudra l'annoncer

Author: Finalement pas CF commit

> | SSHPASSWORD     | Define the password associated with the user name. Cannot be used with the sshcli backend. Warning: using a password is not recommended. Use --ssh-priv-key instead  |                   |             |

Author: ```suggestion
| SSHPASSWORD     | Define the password associated with the user name. Cannot be used with the sshcli backend. Warning: using a password is not recommended. Use a private key instead located in `/var/lib/centreon-engine/.ssh/` or use `--ssh-priv-key` if it's stored elsewhere  |                   |             |
```

> | SSHPASSWORD     | Define the password associated with the user name. Cannot be used with the sshcli backend. Warning: using a password is not recommended. Use a private key instead located in `/var/lib/centreon-engine/.ssh/` or use `--ssh-priv-key` if it's stored elsewhere  |                   |             |

Author: ```suggestion
| SSHPASSWORD     | Define the password associated with the user name. Cannot be used with the sshcli backend. Warning: using a password is not recommended. Use a private key instead and store it in `/var/lib/centreon-engine/.ssh/` or use `--ssh-priv-key` if you stored it elsewhere  |                   |             |
```
